lib/screens/questionaire_screen/questionaire_page_storage.dart

```diff
@@ -261,19 +259,52 @@ class QuestionairePageStorage {
     return patientProfile;
   }
 
+  Future<int?> _getBodyHeight() async {
+    PatientData patientData = const PatientData(
+      patientObjectId: '',
+      doctorObjectId: '',
+    );
+
+    try {
+      dynamic responsePatient = await Backend.getEntry(
+        PatientData.databaseTable,
+        'Patient',
+        Backend.user.objectId!,
+      );
+      dynamic element = responsePatient?.results?.first;
+
+      if (element != null) {
+        patientData = PatientData(
+          bodyHeight: element['BodyHeight'].toDouble(),
+          patientObjectId: element['Patient']['objectId'],
+          doctorObjectId: element['Doctor']['objectId'],
+          objectId: element['objectId'],
+          createdAt: element['createdAt'],
+          updatedAt: element['updatedAt'],
+        );
+      }
+    } catch (e) {
+      rethrow;
+    }
+
+    if (patientData.bodyHeight != null) return patientData.bodyHeight!.toInt();
+
+    return null;
+  }
```

Comment:
You do not need objectId, doctorId ect...
so you can refactor the code, maybe like:

Future<int?> _getBodyHeight() async {
double? bodyHeight;

    try {
      dynamic responsePatientData = await Backend.getEntry(
        PatientData.databaseTable,
        'Patient',
        Backend.user.objectId!,
      );
      dynamic element = responsePatientData?.results?.first;

      bodyHeight = element?['BodyHeight']?.toDouble();
    } catch (e) {
      rethrow;
    }

    if (bodyHeight != null) return bodyHeight.toInt();

    return null;
}

Reply:
Resolved